janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -144,6 +144,48 @@
             ConfigOption.Type.FIXED,
             64);
 
+    ConfigOption<Integer> LOCAL_CORE_CONNECTIONS_PER_HOST = new ConfigOption<>(
+            CQL_NS,
+            "local-core-connection-per-host",
```

Comment:
local-core-connections-per-host? same for the others

Reply:
@analytically I fixed and updated my branch.